bindings/python/tests/tester.py

```diff
@@ -125,6 +125,17 @@ def push(self, val):
         self.stack.push(self.index, val)
 
 
+def test_fdb_transactional_generator(db):
+    try:
+        @fdb.transactional
+        def function_that_yields(tr):
+            yield 0
+        function_that_yields(db)
+        assert fdb.get_api_version() < 620, "Generators post-6.2.0 should throw"
```

Comment:
I don't believe the code has been updated to support API version 620 yet. Were you able to test the new behavior (e.g. by removing the API version guard)?

Reply:
I thought I did this before, but I re-ran and it pointed out a valid, yet different, bug that I think got lost in the spam from when binding tester was more broken than now.  So that's nice.